__npmClientVue_README.md

```diff
@@ -0,0 +1,52 @@
+
```

Comment:
@RHMason, if you want to preview how GitHub renders a Markdown file, you can get there by following these steps:

1. Click the `Files changes` tab on the pull request
2. Scroll to the Markdown file
3. Click the `...` on the top right of the file to expand the menu
4. Click `View file` to see the rendered Markdown

Here's the direct link in case you want to look it over. It looks fine to me.

https://github.com/codeforlansing/cityzen-docs/blob/7801d96f4690ae874a5fa8db5d46d3e86fb63f7c/__npmClientVue_README.md